src/Kunstmaan/AdminBundle/Resources/views/Form/fields.html.twig

```diff
@@ -195,10 +195,15 @@
                                 <i class="fa fa-chevron-down"></i>
                             </button>
                             {% endif %}
+                            {% if obj.children | length > 5 %}
```

Comment:
Would it make sense to make this number configurable?

Reply:
I'm actually not sure how. I can add it to the configuration and then add it to parameters, but you can't inject a parameter into twig, unless its been made available globally.